data/human/pirate jobs.txt

```diff
@@ -3396,3 +3396,21 @@ mission "Republic Assassination [4]"
 		"reputation: Republic" -= 25
 		"reputation: Navy (Oathkeeper)" -= 25
 		"reputation: Pirate" += 10
+
+mission "Slave revolt"
+	landing
+	repeat
+	to offer
+		random < 10
+	on offer
+		require "Slave" 2
+		conversation
+			`	You are flicking through your diary in your cabin, when you hear your door start to open.`
+			`	You look up, one of your slaves is standing in the doorway... holding a large metal bar.`
+			choice
+				`    Grab for a weapon.`
+					goto grab
+			label "grab"
```

Comment:
```suggestion
					
```
Not necessary, it will already go from up to down, if the result of the choice is immediately below it then the goto and label isn't required.

Also not to mention that single-choices are typically not preferred unless a paragraph/text is getting too long(using choice to break it up).